sql-files/main.sql

```diff
@@ -910,6 +910,7 @@ CREATE TABLE IF NOT EXISTS `sc_data` (
   `account_id` int(11) unsigned NOT NULL,
   `char_id` int(11) unsigned NOT NULL,
   `type` smallint(11) unsigned NOT NULL,
+  `tick_total` bigint(20) NOT NULL,
```

Comment:
I'm not really fan of tick_total namming.
I mean a tick is a timepoint, timepoint_total doesn't make much sense in my head.
tick_duration would have been better imo. We want to define an interval of tick for the status ? ok

now does the remaining `tick`, will present a timepoint of the timeout for the skill, or will it be a duration as well ?
where tick < tick_duration